stdlibs/__init__.py

```diff
@@ -6,315 +6,50 @@
 """
 
 __author__ = "John Reese"
+
+import importlib
 import sys
+from typing import Optional
 
 from .__version__ import __version__
+from .py3 import module_names
+
+ALL = "all"
+
+KNOWN_VERSIONS = [
+    "2.3",
+    "2.4",
+    "2.5",
+    "2.6",
+    "2.7",
+    "3.0",
+    "3.1",
+    "3.2",
+    "3.3",
+    "3.4",
+    "3.5",
+    "3.6",
+    "3.7",
+    "3.8",
+    "3.9",
+    "3.10",
+]
+
+
+def stdlib_module_names(version: Optional[str] = None):
```

Comment:
Add return type, `typing.FrozenSet[str]`?

Reply:
Until you enforce this in config, it's going to keep happening... https://github.com/python/mypy/issues/5205